notes/2023/2023-03-23.md

```diff
@@ -0,0 +1,95 @@
+# 2023-03-23 ESLint TSC Meeting Notes
+
+## Transcript
+
+[`2023-03-23-transcript.md`](2023-03-23-transcript.md)
+## Attending
+
+* Nicholas C. Zakas (@nzakas) - TSC
+* Milos Djermanovic (@mdjermanovic) - TSC
+
+@nzakas moderated, and @sam3k took notes.
+
+## Topics
+
+### [feat: no-restricted-imports option added allowImportNames #16196](https://github.com/eslint/eslint/pull/16196)
+
+**Summary:** The PR aims to add an option to the `no-restricted-imports` rule that could possibly conflict with a solution for the accepted issue. This PR keeps getting stale warnings and it looks like it's ready for review. However, we should consider finishing the design in issue [#15261](https://github.com/eslint/eslint/issues/15261) before moving forward with this change.
+
+**Resolution:** We've decided to hold off on merging the PR and update the issue with a summary.
+
+**Action Item:** 
+* @mdjermanovic will summarize the current state of the issue as a comment
+
+### [Change Request: add ability to shard eslint #16726](https://github.com/eslint/eslint/issues/16726)
+
+**Summary:** Request to add a `--shard` to the CLI that will limit which files are linted. This would allow others to more easily implement parallel linting. Jest and Vitest have a similar flag. 
+
+**Resolution:** We've decided to accept a RFC for `--shard` so we can determine whether or not to move forward.
+
+### [Rule Change: [no-loop-func] don’t warn when control flow will always leave the loop #16954](https://github.com/eslint/eslint/issues/16954)
+
+**Summary:** Propose changing `no-loop-func` to exempt a particular pattern.
+
+> @nzakas:
+>
+> We've always said that we want rules to cover the 90% case, and I think this definitely falls within the 10% case.
+>
+> Okay, we've decided to not accept this proposal as the complexity cost isn't worth it.
```

Comment:
I think adding a resolution section to match the rest will make this easier to skim:

```suggestion
**Resolution:** We are not going to accept this proposal as the complexity cost is too high.
```